wandb/proto/wandb_internal.proto

```diff
@@ -46,6 +46,7 @@ message Record {
     RunPreemptingRecord preempting = 23;
     LinkArtifactRecord link_artifact = 24;
     UseArtifactRecord use_artifact = 25;
+    WandbConfigParametersRecord wandb_config_parameters = 26;
```

Comment:
making sure you want to make it a record. records are persistent in the transaction log (aka wandb file)

Reply:
Also i think the name is a bit confusing... is it suppose to be a general interface or only for launch/job building?

Reply:
Yeah agreed, I will make this refer to launch specifically